smalltalksrc/VMMaker/CogARMv8Compiler.class.st

```diff
@@ -5668,7 +5668,7 @@ CogARMv8Compiler >> rewriteCallFullAt: callSiteReturnAddress target: callTargetA
 	^self
 		rewriteFullTransferAt: callSiteReturnAddress
 		target: callTargetAddress
-		expectedInstruction: 16rE12FFF3C
+		expectedInstruction: 16rD63F0200
```

Comment:
`16rD63F0200 aarch64Disassembled  "nil:	blr	x16"`

`16rE12FFF3C` was an old value for ARM32 that never got updated.
But, since that value is used only in an assertion that was failing when running the VM in debug mode, we never noticed it.